### ISSUE TITLE
[Question] Error:  Executable doesn't exist at /home/user/.cache/ms-playwright/ffmpeg-1007/ffmpeg-linux 

### ISSUE THREAD

**Issue by PriyankaSoni1210** [microsoft/playwright#19229]
Hello,

We are using PW version 1.27.1. While trying to connect to selenium-grid implementation using connectOptions in config file:

{
  name: 'chrome',
  use: { wsEndpoint: `wss://${endpoint}/playwright/chrome/playwright-1.27.1`}
}.

The above is working fine for chromium, webkit etc but throwing error for chrome. 
 
Error Msg:
browserContext.newPage: Executable doesn't exist at /home/user/.cache/ms-playwright/ffmpeg-1007/ffmpeg-linux
    ╔═════════════════════════════════════════════════════════════════════════╗
    ║ Looks like Playwright Test or Playwright was just installed or updated. ║
    ║ Please run the following command to download new browsers:              ║
    ║                                                                         ║
    ║     npx playwright install                                              ║
    ║                                                                         ║
    ║ <3 Playwright Team                                                      ║

Tried the following but still getting the same error:
1. Running npx playwright install
2. Updating to version 1.28 is still giving the same error.

Any suggestions on how to solve this?

**Comment by mxschmitt** [comment]
Looks like you don't have locally the 1.27 version and it will fallback to 1.28 which is latest.

If you have `^1.27` inside your package.json, then it will use 1.28 since there is a caret symbol. You can check the Playwright version via: `npx playwright --version`.

**Comment by PriyankaSoni1210** [comment]
Hello, The @playwright/test and playwright versions in package.json are "~1.27.1" and npx playwright --version also confirmed the version to be 1.27.1. 
Although, I did try upgrading to version 1.28 and that didn't solve the problem. 

Some other findings:
1. The issue is only for chrome browser and rest(chromium,webit etc) are working just fine. 
2. If I create a fixture to override the page fixture, I am able to connect to the wsEndpoint and tests are running so it looks like related to @playwright/test runner.

**Comment by mxschmitt** [comment]
This above does not look like Selenium Grid, since you cant connect to a Selenium Grid cluster via connectOptions.

Lets summarise, who launches the Playwright server? Is it e.g. moon?

**Comment by aslushnikov** [comment]
This doesn't look like a Playwright issue. And there's no way we can debug this since there's no repro, so I'll have to close this.

Please feel free to re-submit the issue if you think this is a Playwright bug and you have a repro for us to play with! Thanks!

**Comment by PriyankaSoni1210** [comment]
Hello, We are using Moon, unfortunately I cannot disclose the endpoint. However, I think this is related to @playwright/test(connectOptions) as we are able to reach the endpoint by overriding the page fixture.

**Comment by vania-pooh** [comment]
@PriyankaSoni1210  Yes, for Chrome images we do not install ffmpeg.  https://github.com/aerokube/moon/issues/361
